src/main/java/net/syscon/elite/service/InmateService.java

```diff
@@ -683,6 +687,24 @@ private void validate(final CategoryRejectionDetail detail) {
         return repository.findInmateAliases(bookingId, defaultOrderBy, sortOrder, offset, limit);
     }
 
+    @VerifyBookingAccess
+    public List<SecondaryLanguage> getSecondaryLanguages(final Long bookingId) {
+        return offenderLanguageRepository
+                .findByOffenderBookId(bookingId)
+                .stream()
+                .filter(lang -> lang.getType().equalsIgnoreCase("SEC"))
```

Comment:
Could `lang.getType()` be null?

Reply:
Would suggest this is reversed perhaps...

`"SEC".equalsIgnoreCase(lang.getType())`